src/components/nodeInfoDisplayer.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="info" v-if="percentage">
-    <span>{{percentage}}</span><br/> {{description}}
+    <span>{{percentage}}</span>
+    <br/> {{description}}
+    <br/> ({{current.value}} / {{base}})
```

Comment:
I think having the absolute values (not only %) can be useful. I admit it may also feel a bit too much information. Maybe a solution would be a parameter to decide wether to show absolute values or not?

Another way of displaying the information would be some kind of tooltip on the mouse pointer when hovering over a section.